tests/test_distributions.py

```diff
@@ -197,7 +199,11 @@ def test_single():
         distributions.UniformDistribution(low=1.0, high=1.001),
         distributions.LogUniformDistribution(low=7.3, high=10),
         distributions.DiscreteUniformDistribution(low=-30, high=-20, q=2),
+        distributions.DiscreteUniformDistribution(low=-30, high=-20, q=10),
+        # In Python, "0.3 - 0.2 != 0.1" is True.
+        distributions.DiscreteUniformDistribution(low=0.2, high=0.3, q=0.1),
```

Comment:
@hvy How about this?

```suggestion
        # In Python, "0.8 - 0.7 == 0.1` is False but "0.3 - 0.2 != 0.1" is True.
        distributions.DiscreteUniformDistribution(low=0.7, high=0.8, q=0.1),
        distributions.DiscreteUniformDistribution(low=0.2, high=0.3, q=0.1),
```

Reply:
Should it not be "and" instead of "but"? And a detail but I think you places a backtick by mistake 🙂

Reply:
Thank you for pointing it out. I updated a comment.